packages/docs/src/components/content/MaturityChecklist/MaturityChecklist.tsx

```diff
@@ -1,93 +1,222 @@
-        Component designed to work in all responsive breakpoints.
-      </MaturityChecklistItem>
-      <MaturityChecklistItem title="Spanish translations" status={props.spanish}>
-        Includes Spanish translations for default text content.
-      </MaturityChecklistItem>
-    </ul>
-
-    {/* <h3>Design</h3>
-    <ul className="ds-c-list--bare">
-      <MaturityChecklistItem title="Sketch UI-kit" status={props.completeUiKit}>
-        Includes all Sketch symbols for defined options.
-      </MaturityChecklistItem>
-      <MaturityChecklistItem title="Responsive" status={props.responsiveUiKit}>
-        All Sketch symbols designed for small and large breakpoints.
-      </MaturityChecklistItem>
-    </ul> */}
-
-    <h3>Tokens</h3>
-    <ul className="ds-c-list--bare">
-      <MaturityChecklistItem title="Code" status={props.tokensInCode}>
-        Tokens implemented in code.
-      </MaturityChecklistItem>
-      <MaturityChecklistItem title="Design" status={props.tokensInSketch}>
-        Tokens implemented in the Sketch.
-      </MaturityChecklistItem>
-    </ul>
-  </section>
-);
+const MaturityChecklist = (props: MaturityChecklistProps) => {
+  const criteria = [
+    {
+      title: 'Accessibility',
+      items: [
+        {
+          title: 'Color',
+          description: 'Meets AA color contrast standards for accessibility and color blindness.',
+          status: props.items['color'],
+        },
+        {
+          title: 'Forced Colors Mode (FCM)',
+          description: 'While using FCM the components text is legible and improves readability.',
+          status: props.items['forcedColors'],
+        },
+        {
+          title: 'WCAG 2.1 Level AA Conformance',
+          description: 'All Axe checks for WCAG AA compliance have passed.',
+          status: props.items['a11yStandards'],
+        },
+        {
+          title: 'Screen readers',
+          description:
+            'VoiceOver, NVDA, and JAWS screen readers provide concise communication and interaction.',
+          status: props.items['screenReaders'],
+        },
+        {
+          title: 'Keyboard navigation',
+          description: 'Component is fully navigable with a keyboard.',
+          status: props.items['keyboardNavigable'],
+        },
+      ],
+    },
+    {
+      title: 'Code',
+      items: [
+        {
+          title: 'Storybook',
+          description: 'Component has stories to cover all defined props.',
+          status: props.items['storybook'],
+        },
+        {
+          title: 'Responsive',
+          description: 'Component designed to work in all responsive breakpoints.',
+          status: props.items['responsive'],
+        },
+        {
+          title: 'Spanish translations',
+          description: 'Includes Spanish translations for default text content.',
+          status: props.items['spanish'],
+        },
+      ],
+    },
+    // {
```

Comment:
design criteria was commented out in original layout for some reason. would be nice to remove or add more context for why it's commented out so it can be addressed at some point.